R/dataset_vignette.R

```diff
@@ -0,0 +1,74 @@
+#' Output a dataset in a vignette in the admiral format
+#'
+#' Output a dataset in a vignette with the pre-specified format
+#' By default, the variables specified in `display_vars`
```

Comment:
The end of the sentence is missing.

Reply:
I have removed the sentence because the details are given for the `display_vars` arg.